swift-sdk/Internal/RequestCreator.swift

```diff
@@ -449,7 +449,7 @@ struct RequestCreator {
     
     private func createPostRequest(path: String, body: [AnyHashable: Any]? = nil) -> PostRequest {
         PostRequest(path: path,
-                    args: [JsonKey.Header.apiKey: apiKey],
+                    args: nil,
```

Comment:
I don't think we need `apiKey` for this class anymore. This seems to be the only place apiKey is being used. We should probably remove it.